content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -211,7 +211,9 @@ kubectl get pod private-reg
 ```
 
 {{< note >}}
-In case the Pod fails to start with the status `ImagePullBackOff`, view the Pod events:
+Please ensure that the Pod or Deployment, and so on, created within a particular namespace contains the necessary secret in that same namespace.
```

Comment:
Secret should be in upper case. I would actually write:
```suggestion
To use image pull secrets for a Pod (or a Deployment, or other object that
has a pod template that you are using), you need to make sure that the appropriate
Secret does exist in the right namespace. The namespace to use is the same
namespace where you defined the Pod.
````

Reply:
Thanks @sftim